<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>194</width>
    <height>278</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777</width>
    <height>16777</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>174</width>
     <height>218</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="Clear_Button">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="Add_Button">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="Sub_Button">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="Back_Button">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="Seven_Button">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="Eight_Button">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="Nine_Button">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="Mul_button">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="Four_Button">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="Five_Button">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="Six_Button">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="Div_Button">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="One_Button">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="Two_Button">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Three_Button">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="Left_Button">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>(</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="Zero_Button">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="Right_Button">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" rowspan="2">
     <widget class="QPushButton" name="Equal_Button">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>80</height>
       </size>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
